middleware/middleware-1.3.11.md

```diff
@@ -0,0 +1,63 @@
+
+## New SCU: SwissbitCloud (Sandbox)
+To increase our portfolio of supported cloud TSEs, we added a new SCU package that enables our Middleware users to include the newly available Swissbit Cloud TSE. 
+
+Like the _DeutscheFiskal_ SCU, this new package automatically downloads and initializes the Fiscal Cloud Connector (the backend web service that is required to operate this TSE). If required, it's also possible to bypass this functionality and manually install this service by passing the `FccDirectory` parameter.
+
+While this TSE is running smoothly in our tests so far, there are some limitations, both technically and due to the vendor's business model:
+- The SwissbitCloud TSE was only certified for Windows 10 and Ubuntu 20.04 so far. Although the producer is working on additional certifications for other OSs, it can meanwhile officially only be operated on these systems. We're in continuous communication with Swissbit and will notify our users as soon as there are only news about this topic.
+- Due to the pricing model (pricing is calculated per client, not per TSE), our usual rollout scenarios are limited. We are working on updating our [documentation](https://docs.fiskaltrust.cloud/doc/productdescription-de-doc/for-posdealers/02-pre-sales/rollout-scenarios.html) to reflect these limitations.
+
+## Stability improvements for hardware TSEs
+In this version, we implemented various changes in our hardware SCU packages that should greatly improve the overall stability when working with **Swissbit, CryptoVision or Epson** TSEs. 
+
+We especially focused on the stability of log exports and recurring operations like the self-tests and time updates, which should now also occur less often (and therefore reduce the number of consumed signatures).
+
+## Performance improvements for hardware TSEs
+While introducing the stability improvements mentioned above, we also worked hard to increase the performance of the **Swissbit, CryptoVision and Epson** TSEs. This is mostly reflecting in the performance of the log file exports, which - depending on the TSE - should now consume up to 50% less time.
+
+## New feature: Opt-out of telemetry collection in online mode
+The fiskaltrust Middleware collects usage and especially error telemetry, using Microsoft Application Insights. Previously, it was only possible to opt-out of the telemetry collection by using the Middleware in the offline mode, which comes with several disadvantages (e.g. no data upload). 
+
+We therefore introduced a new Launcher parameter that completely deactivates all logging to Application Insights, `--telemetry-optout`.
+
+Please keep in mind that using this will prevent us from actively reacting to issues in the respective customer installations, as we will be completely dependent on receiving inquiries and log files in those case.
```

Comment:
We could add sth regarding the Metrics section here, that in case this is disabled they won´t be able to see that in the portal

Reply:
Thanks, added it :+1: